### ISSUE TITLE
Object overrides and Panda World overrides

### ISSUE THREAD

**Issue by Gymnasiast** [OpenRCT2/Localisation#2005]
Updated strings from https://github.com/OpenRCT2/OpenRCT2/commit/14512e72a3a145726c02d8afc123566e4e95d522

The following overrides have been moved to JSON objects. Remove them from your language file and make so their descriptions should be present in the [objects repository](https://github.com/OpenRCT2/objects/tree/master/objects/official)

```diff
-[XXBBBR01]
-STR_NAME    :Base Block
-
-[TTRFTL02]
-STR_NAME    :Roof
-
-[TTRFTL03]
-STR_NAME    :Roof
-
-[TTRFTL04]
-STR_NAME    :Roof
-
-[TTRFTL07]
-STR_NAME    :Roof
-
-[TTRFTL08]
-STR_NAME    :Roof
-
-[MG-PRAR]
-STR_NAME    :Wall with Passageway
```

On the other hand, the commit adds a description for Panda World, the only official DLC scenario for RCT2. Add it below your override for `Rock 'n' Roll - Rock 'n' Roll`

```
###############################################################################
## Official Custom Content Scenarios
###############################################################################
<Panda World>
STR_SCNR    :Panda World
STR_PARK    :Panda World
STR_DTLS    :Add more rides and attract more people to this panda-themed park
```

Updated in:
- [ ] ar-EG
- [ ] ca-ES
- [ ] cs-CZ
- [ ] da-DK
- [ ] de-DE
- [ ] en-US
- [ ] eo-OO
- [ ] es-ES
- [ ] fr-FR
- [ ] hu-HU
- [ ] it-IT
- [ ] ja-JP
- [ ] ko-KR
- [ ] nb-NO
- [ ] nl-NL
- [ ] pl-PL
- [ ] pt-BR
- [ ] sv-SE
- [ ] tr-TR
- [ ] zh-CN
- [ ] zh-TW

Notification when new strings arrive:
@Omaranwa @J0anJosep @octaroot @LPSGizmo @danidoedel @Wuzzy2 @tellovishous @JasperGray @Wirlie @JoelTroch @anon569 @LucaRed @AaronVanGeffen @telk5093 @Goddesen @renansimoes @Tupaschoal @Nubbie @izhangfei @daihakken @marcinkunert

**Comment by frenchiveruti** [comment]
No way to find these strings that you are saying either on the es-ES.txt or the .json file.  
How am I supposed to know they're present?

**Comment by frenchiveruti** [comment]
Also I can only apply a partial solution to this issue, I can do the panda world part, but I can't find these:
  

> ```diff
> -[XXBBBR01]
> -STR_NAME    :Base Block
> -
> -[TTRFTL02]
> -STR_NAME    :Roof
> -
> -[TTRFTL03]
> -STR_NAME    :Roof
> -
> -[TTRFTL04]
> -STR_NAME    :Roof
> -
> -[TTRFTL07]
> -STR_NAME    :Roof
> -
> -[TTRFTL08]
> -STR_NAME    :Roof
> -
> -[MG-PRAR]
> -STR_NAME    :Wall with Passageway
> ```

anywhere.

**Comment by AaronVanGeffen** [comment]
@frenchiveruti They've been removed from the interface language files, and are now in the [object language files](https://github.com/OpenRCT2/Localisation/tree/master/objects).

**Comment by frenchiveruti** [comment]
> 
> 
> @frenchiveruti They've been removed from the interface language files, and are now in the [object language files](https://github.com/OpenRCT2/Localisation/tree/master/objects).

I understood that fact, the problem is that how do I know which one is each, there's like 20 "Roof" strings in the json files.

**Comment by Gymnasiast** [comment]
> how do I know which one is each, there's like 20 "Roof" strings in the json files.

By looking at the line that immediately proceeds it.

```
  "other.ttrftl02": {
    "reference-name": "Roof",
    "name": "Tejado"
  },
```

**Comment by frenchiveruti** [comment]
Okay that solves it, thanks, I see now that it's inside "other.xxxx" what I should be looking at.  
And it seems that I have a lot of strings to correct there ha. Some translations are... yuck... 
Thanks

**Comment by frenchiveruti** [comment]
Hijacking this conversation for this question:
Can I do something like:
```json
  "rct2.tt.artdec22": {
    "reference-name": "Art Deco Wall with Railings",
    "name": "Muro Art Decó con balaustrada"
  },
  "rct2.tt.artdec23": {
    "reference-name": "Art Deco Wall with Railings",
    "name": "Muro Art Decó con balaustrada"
  },
  "rct2.tt.artdec21": {
    "reference-name": "Art Deco Wall with Railings",
    "name": "Muro Art Decó con balaustrada"
  },
```

into:
```json
  "rct2.tt.artdec21","rct2.tt.artdec22","rct2.tt.artdec23": {
    "reference-name": "Art Deco Wall with Railings",
    "name": "Muro Art Decó con balaustrada"
  },
```
I have no idea how json works, I'm just curious if it works like CSS when it comes to using the same qualities for different objects.

**Comment by AaronVanGeffen** [comment]
No, that won't work. You cannot combine entries, sorry. (We're not happy about the duplication, either…)

**Comment by frenchiveruti** [comment]
Okay no problem it was just a matter of beautifying. I guess it's a thing of JSON.
